ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/diffbased/verkle/LeafBuilder.java

```diff
@@ -49,9 +49,13 @@ public void generateAccountKeyForRemoval(final Address address) {
     keysForRemoval.add(trieKeyFactory.basicDataKey(address));
   }
 
+  public void generateCodeHashKeyForRemoval(final Address address) {
+    keysForRemoval.add(trieKeyFactory.codeHashKey(address));
+  }
+
   public void generateCodeKeysForRemoval(final Address address, final Bytes code) {
     keysForRemoval.add(trieKeyFactory.basicDataKey(address));
```

Comment:
this one seems to be a problem no ? if we remove the code but not the account, this one will remove the basic data leaf ?